modules/network-flow-matrix.adoc

```diff
@@ -79,7 +79,7 @@ In addition to the base network flows, the following matrix describes the ingres
 .{sno-caps} on bare metal
```

Comment:
can we also change the title here?
we are actually documenting `PlatformType=none` here
https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/installing_on_any_platform/installing-platform-agnostic